### ISSUE TITLE
Tracker implementation/documentation incorrectly refers to width and height settings as "max"

### ISSUE THREAD

**Issue by rjhowell44** [prominenceai/deepstream-services-library#537]
Cleanup from early mistakes

**Comment by rjhowell44** [comment]
The two constructors will be updated to reflect the correct parameter names and description 

```C
/**
 * @brief creates a new, uniquely named KTL Tracker object
 * @param[in] name unique name for the new Tracker
 * @param[in] width output frame width for GIEs and TISs to work on
 * @param[in] height output frame height for GIEs and TIS to work on
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_TRACKER_RESULT otherwise
 */
DslReturnType dsl_tracker_ktl_new(const wchar_t* name, uint width, uint height);

/**
 * @brief creates a new, uniquely named IOU Tracker object
 * @param[in] name unique name for the new Tracker
 * @param[in] config_file fully qualified pathspec to the IOU Lib config text file
 * @param[in] width output frame width for GIEs and TISs to work on
 * @param[in] height output frame height for GIEs and TIS to work on
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_TRACKER_RESULT otherwise
 */
DslReturnType dsl_tracker_iou_new(const wchar_t* name, 
    const wchar_t* config_file, uint width, uint height);

```
The `dsl_tracker_max_dimensions_get` and `dsl_tracker_max_dimensions_get` will be updated to the following.
```C
/**
 * @brief returns the current maximum frame width and height settings for the named IOU Tracker object
 * @param[in] name unique name of the Tracker to query
 * @param[out] width output frame width for GIEs and TISs to work on
 * @param[out] height output frame height for GIEs and TIS to work on
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_TRACKER_RESULT otherwise
 */
DslReturnType dsl_tracker_dimensions_get(const wchar_t* name, uint* width, uint* height);

/**
 * @brief sets the maximum frame width and height settings for the named IOU Tracker object
 * @param[in] name unique name of the Tracker to update
 * @param[in] width output frame width for GIEs and TISs to work on
 * @param[in] height output frame height for GIEs and TIS to work on
 * @return DSL_RESULT_SUCCESS on success, DSL_RESULT_TRACKER_RESULT otherwise
 */
DslReturnType dsl_tracker_dimensions_set(const wchar_t* name, uint width, uint height);
```

**Comment by rjhowell44** [comment]
Merged into v0.16.alpha
